javascript-arrays/main.js

```diff
@@ -0,0 +1,42 @@
+var colors;
+colors = ['red', 'white', 'blue'];
+
+console.log('Value of colors:', colors);
+console.log('Type of colors:', typeof colors);
+
+console.log('Value of colors [0]:', colors[0]);
+console.log('Value of colors [1]:', colors[1]);
+console.log('Value of colors [2]:', colors[2]);
+
+var americaIs = 'America is';
+americaIs += ' ' + colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.';
+console.log(americaIs);
+
+colors[2] = 'green';
+console.log('Value of colors [2]:', colors[2]);
+
+var mexicoIs = 'Mexico is';
+mexicoIs += ' ' + colors[0] + ', ' + colors[1] + ', and ' + colors[2] + '.';
+console.log(mexicoIs);
+
+console.log('Value of colors:', colors);
+
+var students;
+students = ['Anne', 'Murphy', 'Michael', 'Sarah'];
```

Comment:
This isn't incorrect, however you can declare a variable and do an assignment on the same line.
```
var students = ['Anne', 'Murphy', 'Michael', 'Sarah'];